.gitignore

```diff
@@ -61,4 +61,6 @@ examples/public/
 documentation/.docusaurus
 documentation/generated
 documentation/docs/api
-/react-components/stories/public/cadModel
+/react-components/stories/public/cadModel
+
+**/install-state.gz
```

Comment:
:monocle_face: What's this?

Reply:
<img width="969" alt="image" src="https://github.com/user-attachments/assets/5a73a737-f17a-469b-91a3-414dca1e60c0">

https://yarnpkg.com/getting-started/qa#:~:text=yarn%2Finstall%2Dstate.,your%20workspaces%20all%20over%20again. :)